scvid/train/training_plan.py

```diff
@@ -65,10 +65,11 @@ class DummyTrainingPlan(pl.LightningModule):
     def __init__(self, module: torch.nn.Module):
         super().__init__()
         self.module = module
-        self.automatic_optimization = False
```

Comment:
Please update Docstring for `DummyTrainingPlan`:  it'snot just for `OnePassMeanVarStd` -- it's a training plan that does not perform any actual optimization. It could be used for cases where only the forward pass is intended (e.g. for direct inference, calculating sufficient statistics, EM algorithms, etc.).